CMakeLists.txt

```diff
@@ -226,21 +226,18 @@ endif()
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${_gRPC_C_CXX_FLAGS}")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_gRPC_C_CXX_FLAGS}")
 
+if(_gRPC_PLATFORM_MAC)
+  # some C++11 constructs not supported before OS X 10.10
+  set(CMAKE_OSX_DEPLOYMENT_TARGET 10.10)
```

Comment:
I don't see an immediate problem with always setting `CMAKE_OSX_DEPLOYMENT_TARGET` but it's something that we might need to make optional at some point. Probably fine to leave as is for now (can you try manually building with cmake on MacOS Big Sur just to verify it works fine?).

Reply:
This has been set to all other OSX targets so I don't think it would be a problem. https://github.com/grpc/grpc/blob/35569cd544f02967ba59ef2886e7b5ae6880405b/gRPC-C%2B%2B.podspec#L38

I tested building gRPC on OSX Big Sur & Intel and it worked.

Reply:
Sg, thanks for testing!